common/dpsl.ts

```diff
@@ -355,7 +355,7 @@ export interface RunSmartctlCheckRequest {
   percentage_used_threshold?: number;
 }
 
-export type DpslTypes =
+export type TelemetryUnion =
```

Comment:
I'm not sure what this is a union of by the naming.

Can we be more specific? 

If it is the possible telemetry types (e.g. cpu, memory, etc), lets call it `TelemetryTypeUnion`

if it is the possible telemetry results (e.g. CpuResult, MemoryResult), lets call it `TelemetryResultUnion`

Reply:
Since it is the union of some telemetry info, it is renamed to TelemetryInfoUnion.